packages/sdk/tasks/deposit-eth.ts

```diff
@@ -70,7 +84,14 @@ task('deposit-eth', 'Deposits WETH9 onto L2.')
     )
 
     const l2ChainId = await l2Signer.getChainId()
-    const contractAddrs = CONTRACT_ADDRESSES[l2ChainId]
+    let contractAddrs = CONTRACT_ADDRESSES[l2ChainId]
```

Comment:
If its easier to not use the hh deploy artifacts I understand but then this becomes very tied to the way we do it in CI instead of being able to work against live networks as well

Reply:
I don't see why this breaks against live networks. All this code does is let me pass in a JSON with addresses in it rather than using the contract addresses hardcoded in the SDK. We'd have to traverse the HH deploy dir and create this JSON anyway to get the `contracts` arg the SDK expects.